src/views/Records/Record.vue

```diff
@@ -2,6 +2,8 @@
   <div
     class="standard container-fluid"
   >
+
+    <div v-if="$store.state.users.userLoggedIn" class="alert alert-success"> You are logged in.</div>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['class' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=5156911)